app/views/content/non-uk-teachers/fees-and-funding-for-non-uk-trainees.md

```diff
@@ -2,7 +2,7 @@
 title: "Fees and financial support for non-UK trainee teachers"
 subcategory: If you want to train to teach
 description: |-
-  Learn more about funding and financial support for non-UK students, including details of the international relocation payment.
+  Learn more about bursaries and scholarships for non-UK students.
```

Comment:
@laurahermionetennant92 sorry just looking at this, should it be non-UK trainee teachers rather than non-UK students?

Reply:
Yes I think it should...